src/config.js

```diff
@@ -1,15 +1,15 @@
 const config = {
     // basename: only at build time to set, and Don't add '/' at end off BASENAME for breadcrumbs, also Don't put only '/' use blank('') instead,
     // like '/berry-material-react/react/default'
-    basename: '/free',
+    basename: '',
     defaultPath: '/dashboard/default',
```

Comment:
Could you also create a JIRA issue for creation of a landing page? (check if there is one already), and in the description mention that we'll have to change defaultPath in config.js to route to that landing page (instead of the dashboard)